Tringo.WebApp/Controllers/FlightsController.cs

```diff
@@ -46,17 +46,20 @@ public class FlightsController : ControllerBase
                 .ToList();
             var airportsIatas = relatedAirports.Select(a => a.IataCode);
 
+            if (allFlights is null || allAirports is null)
```

Comment:
All airports can't be null - if null, something is really wrong with the service

Maybe it's instead make sense to check if !allFlights.Any()

3.why do you check for null using 'is', but not '==' ??

Reply:
1. It throws null reference errors, I'm addressing it
 (Search for flights from Newcastle)

it's just for readability and is syntactic sugar really

Reply:
1. allAirports won't even be null unless there's a critical bug in the code
2. It does make sense to add it to avoid other filters applied
3. Idk, must be a matter of personal habit, alright

4. Why would you add empty mock files?

Reply:
I removed allAirport check. 
4. Them files automatically generated on every run.